site/source/user-manual/backups/backup-create.rst

```diff
@@ -45,41 +25,9 @@ Backup Using LAN Shared Folder
     .. figure:: /_static/images/config/embassy_backup3.png
         :width: 60%
 
-#. When the backup is complete you will receive a notification in the Notifications tab, and you will see the date and time of your most recent backup updated under "Backups" in the Embassy tab.
+After backup, you will receive a notification that the backup was successful, and you will see the latest backup date and time appear on the Embassy tab.
```

Comment:
I prefer the old copy, especially since the backup may not be successful. Maybe there's a 3rd option.